Index: cell_based/src/simulation/OffLatticeSimulation.cpp
===================================================================
--- cell_based/src/simulation/OffLatticeSimulation.cpp	(revision 25767)
+++ cell_based/src/simulation/OffLatticeSimulation.cpp	(working copy)
@@ -77,7 +77,7 @@
         // All classes derived from AbstractOffLatticeCellPopulation are covered by the above (except user-derived classes),
         // i.e. if you want to use this class with your own subclass of AbstractOffLatticeCellPopulation, then simply
         // comment out the line below
-        NEVER_REACHED;
+//        NEVER_REACHED;
     }
 }
 
Index: mesh/src/common/NodeAttributes.cpp
===================================================================
--- mesh/src/common/NodeAttributes.cpp	(revision 25767)
+++ mesh/src/common/NodeAttributes.cpp	(working copy)
@@ -115,10 +115,10 @@
 template<unsigned SPACE_DIM>
 void NodeAttributes<SPACE_DIM>::SetRadius(double radius)
 {
-    if (radius < 0.0)
-    {
-        EXCEPTION("Trying to set node attributes mRadius to a negative value.");
-    }
+//    if (radius < 0.0)
+//    {
+//        EXCEPTION("Trying to set node attributes mRadius to a negative value.");
+//    }
 
     mRadius = radius;
 }
